salt/client/netapi.py

```diff
@@ -45,12 +43,17 @@ def __getstate__(self):
         }
 
     def run(self):
+        if self.log_queue is not None:
+            salt.log.setup.set_multiprocessing_logging_queue(self.log_queue)
+        if self.log_queue_level is not None:
+            salt.log.setup.set_multiprocessing_logging_level(self.log_queue_level)
+        salt.log.setup.setup_multiprocessing_logging(self.log_queue)
```

Comment:
Additional information can be found in the initial PR: #52471
I've tried to see if this change is effective on top of 3000.2 and it is. Without this change, salt-api is leaking a huge amount of memory. I didn't dig more than this since the manual testing shown me the huge difference.